README.md

```diff
@@ -190,18 +206,11 @@ Stefan Tomanek has a nice writeup about using Vexim in Debian, but that article
 #### Site Admin:
 In order to add and delete domains from the database, you need to have a "site admin". This user can create the initial postmaster users for the individual domains.
 
-The default username and password for the siteadmin, are:
-* **Username:** siteadmin
-* **Domain value:** (blank)
-* **Password:** CHANGE
-
-The password is case sensitive. You are strongly advised to log in and change it as soon as you get a chance. :-)
+The `siteadmin`-user has been created before with an auto-generated password (see mysql-section), use it here to login. The password is case sensitive. You are advised to change it when you first log in.
```

Comment:
This user has been created along with the database (refer to the Databases and authentication section above), use it here to log in. The password is case sensitive. You are advised to change it when you first log in.

And I think you can merge this paragraph with the previous one.